src/commands/trigger.cpp

```diff
@@ -0,0 +1,104 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// trigger.cpp
+//
+// Identification: src/commands/trigger.cpp
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+
+#include "commands/trigger.h"
+#include "parser/pg_trigger.h"
+
+namespace peloton {
+namespace commands {
+Trigger::Trigger(const peloton::planner::CreatePlan& plan) {
+  trigger_name = plan.GetTriggerName();
+  trigger_funcname = plan.GetTriggerFuncName();
+  trigger_args = plan.GetTriggerArgs();
+  trigger_columns = plan.GetTriggerColumns();
+  trigger_when = plan.GetTriggerWhen();
```

Comment:
I am confused about what "trigger_when" is for: is it the trigger time? Can more comment be added for such naming?